### PR TITLE
Security Hub

### PR DESCRIPTION
**Pull request by paulschwarzenberger**
- Enable Security Hub in London region with CIS benchmark standard
- Move conditional statement for DNS delegation from modules
- Move conditional statement for manual creation of hosted zones from modules

### CODE REVIEW

File: ses/main.tf [hunk 1]
```diff
@@ -10,23 +10,23 @@ resource "aws_route53_record" "amazonses_verification_record" {
   records = [aws_ses_domain_identity.email_domain.verification_token]
 }
 
-# include environment in conditional statement until DNS delegation is in place
+# implement this resource after DNS delegation is in place to avoid terraform errors
 resource "aws_ses_domain_identity_verification" "email_verification" {
-  count  = var.environment_full_name == "production" ? 0 : 1
+  count  = var.domain == true ? 0 : 1
   domain = aws_ses_domain_identity.email_domain.id
```
Comment (MancunianSam) [inline ses/main.tf]: The domain variable is a string set to nationalarchives.gov.uk as far as I can see, should we compare it to `true` like this

File: ses/variables.tf [hunk 2]
```diff
@@ -18,4 +18,9 @@ variable "email_address" {
 
 variable "hosted_zone_id" {
   description = "Hosted zone ID"
-}
\ No newline at end of file
+}
+
```
Comment (MancunianSam) [inline ses/variables.tf]: This is the same as I said on the other pull request but I'm not sure what this variable and the `manual_creation` one is doing.

File: ses/main.tf (hunk 1 above)
Comment (paulschwarzenberger) [reply ses/main.tf]: thanks, corrected

File: ses/variables.tf (hunk 2 above)
Comment (paulschwarzenberger) [reply ses/variables.tf]: thanks, corrected

**Pull request merged**